linux/Platform.c

```diff
@@ -107,6 +108,22 @@ static time_t Platform_Battery_cacheTime;
 static double Platform_Battery_cacheLevel = NAN;
 static ACPresence Platform_Battery_cacheIsOnAC;
 
+void Platform_init(void) {
+   if (access(PROCDIR, R_OK) != 0) {
+      fprintf(stderr, "Error: could not read procfs (compiled to look in %s).\n", PROCDIR);
+      exit(1);
+   }
```

Comment:
Isn't this check common to all platforms? Also maybe have `Platform_init` return `bool` and keep exitting to `htop.c`.

Reply:
Nah, PROCDIR is only used on Linux.

Reply:
Oh, also, other functions in htop.c (like arg parsing) exit at the point of failure, so I'd rather leave it as-is for now.